packages/frontend/src/context/Unirep.ts

```diff
@@ -24,6 +24,7 @@ export class UnirepConfig {
     commentReputation = 0
     airdroppedReputation = 0
     attesterId = 0
+    defaultAirdropKarma = 30
```

Comment:
I think it is better to be called `subsidy`
since there is another one called `airdroppedReputation` and actually they are different meaning in the subsidy system

and it can be loaded with `this.unirepSocial.subsidy()`
like here
https://github.com/Unirep/Unirep-Social/blob/b1ffa5ca1de911c1f9356e62b782ff43ed67b7a7/packages/frontend/src/context/Unirep.ts#L58-L68

and we should load both `subsidy` and `airdroppedReputation`

so it can replace
https://github.com/Unirep/Unirep-Social/blob/b1ffa5ca1de911c1f9356e62b782ff43ed67b7a7/packages/frontend/src/context/User.ts#L249
and
https://github.com/Unirep/Unirep-Social/blob/b1ffa5ca1de911c1f9356e62b782ff43ed67b7a7/packages/frontend/src/context/User.ts#L285

both are the same `this.unirepConfig.subsidy`

Reply:
because it is named `DEFAULT_AIRDROPPED_KARMA` in `backend/src/routes/AirdropRouter.ts`, so I think it might be the same thing? Just wanna make sure if I take things wrong -- what I want to declare here is **the maximum amount of airdrop an user can get after UST** , I think it's different from `subsidy`? For example, the user can have 30 subsidy each epoch, while if the user has negRep, he/she can only get maximum 20 defaultAirdroppedKarma each epoch.
So I'm saying that, I don't understand you at this part.
Also, I really don't know what `airdroppedReputation` stands for, in `Unirep.ts`. If this stands for the **maximum airdrop amount each user could get after ust**, then I'll just remove the `defaultAirdropKarma`.

Reply:
> because it is named `DEFAULT_AIRDROPPED_KARMA` in `backend/src/routes/AirdropRouter.ts`

Because we didn’t remove it after change to subsidy
So the `DEFAULT_AIRDROPPED_KARMA` is the old sign up airdrop 
But now the neg rep airdrop is `subsidy` itself 
So the `DEFAULT_AIRDROPPED_KARMA=0` 
We should remove it in the next version(when we deploy a new contract

> the maximum amount of airdrop an user can get after UST

Now is subsidy itself
If you think it should be different then we should change the contract (in the next contract deployment